packages/core/src/controllers/pairing.ts

```diff
@@ -397,6 +398,16 @@ export class Pairing implements IPairing {
       const { message } = getInternalError("MISSING_OR_INVALID", `pair() uri#symKey`);
       throw new Error(message);
     }
+    if (uri?.expiryTimestamp) {
+      const expiration = toMiliseconds(uri?.expiryTimestamp);
+      if (expiration < Date.now()) {
```

Comment:
Probably strictly theoretical rather than realistic, but I think there's an edge case here where:
* `toMiliseconds` simply adds zeroes to the end of seconds-based `expiration`
* `Date.now()` has non-zero values in the last 3 digits

Either way I think we can ignore because that only applies within the same second :)